### PR TITLE
Added dateTime for a single SoftwareUnitTestResult

### PR DESCRIPTION
**Pull request by AbhaMoitra**
Added dateTime to just UTR-1-1-1 to illustrate syntax and test RACK workflow. This works with dev version of RACK dated 1/28/2021/

### CODE REVIEW

File: RACK-Ontology/OwlModels/SACM-S.owl [hunk 1]
```diff
@@ -18,12 +18,12 @@
     <rdfs:comment xml:lang="en">This ontology was created from a SADL file 'SACM-S.sadl' and should not be directly edited.</rdfs:comment>
   </owl:Ontology>
   <owl:Class rdf:ID="ASSERTED_CONTEXT">
-    <rdfs:comment xml:lang="en">Asserted Context can be used to declare that the artifact cited by an ArtifactReference(s) provides the context for the
+    <rdfs:comment xml:lang="en">Asserted Context can be used to declare that the artifact cited by an ArtifactReference(s) provides the context for the&#xD;
 interpretation and scoping of a Claim or ArgumentReasoning element.</rdfs:comment>
```
Comment (kquick) [inline RACK-Ontology/OwlModels/SACM-S.owl:22]: What is this change?  It doesn't seem to correspond to a change in any .sadl file, and it doesn't seem like a change we want?

File: RACK-Ontology/OwlModels/SACM-S.owl [hunk 2]
```diff
@@ -18,12 +18,12 @@
...
     <rdfs:subClassOf rdf:resource="PROV-S#ENTITY"/>
   </owl:Class>
   <owl:Class rdf:ID="ASSERTED_INFERENCE">
-    <rdfs:comment xml:lang="en">Asserted Inference association records the inference that a user declares to exist between one or more Assertion (premise) and
+    <rdfs:comment xml:lang="en">Asserted Inference association records the inference that a user declares to exist between one or more Assertion (premise) and&#xD;
```
Comment (kquick) [inline RACK-Ontology/OwlModels/SACM-S.owl:26]: Another odd character introduction.

File: RACK-Ontology/OwlModels/SACM-S.owl [hunk 3]
```diff
@@ -32,7 +32,7 @@ another Assertion (conclusion).</rdfs:comment>
     <rdfs:subClassOf rdf:resource="PROV-S#ENTITY"/>
   </owl:Class>
   <owl:Class rdf:ID="ARGUMENT_PACKAGE">
-    <rdfs:comment xml:lang="en">Argument Package is the containing element for a structured argument represented using the SACM Argumentation
+    <rdfs:comment xml:lang="en">Argument Package is the containing element for a structured argument represented using the SACM Argumentation&#xD;
```
Comment (kquick) [inline RACK-Ontology/OwlModels/SACM-S.owl:35]: Another odd character introduction.

File: Turnstile-Ontology/99-Utils/Data/SoftwareUnitTestResult.csv [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-"Passed","SoftwareUnitTestRun1","UTC-1-1","UTR-1-1-1"
+"identifier_TEST_STATUS","generatedAtTime_SoftwareUnitTestResult","identifier_SoftwareUnitTestExecution","identifier_SoftwareUnitTest","identifier_SoftwareUnitTestResult"
+"Failed",null,"SoftwareUnitTestRun2","UTC-1-4","UTR-1-4-2"
+"Passed",null,"SoftwareUnitTestRun2","UTC-1-3","UTR-1-3-2"
+"Passed",null,"SoftwareUnitTestRun2","UTC-1-2","UTR-1-2-2"
+"Passed",null,"SoftwareUnitTestRun2","UTC-1-1","UTR-1-1-2"
+"Failed",null,"SoftwareUnitTestRun1","UTC-1-4","UTR-1-4-1"
+"Passed",null,"SoftwareUnitTestRun1","UTC-1-3","UTR-1-3-1"
+"Passed",null,"SoftwareUnitTestRun1","UTC-1-2","UTR-1-2-1"
+"Passed","Wed Mar 22 20:00:00 EDT 2017","SoftwareUnitTestRun1","UTC-1-1","UTR-1-1-1"
```
Comment (kquick) [inline Turnstile-Ontology/99-Utils/Data/SoftwareUnitTestResult.csv:9]: This timestamp doesn't seem to be the same as the one in CounterApplicationUnitTesting.sadl, even allowing for timezone conversions?

**Comment by AbhaMoitra** [comment]
@kquick : good catches! Not sure why the weird characters got introduced. Looking into the dateTime issue.

**Comment by glguy** [comment]
I'm working on update this PR against ontology changes. I'll push the updates soon.

There's definitely something wrong with the way the date times are being rendered to CSV, however. We should fix that, but it probably doesn't need to block this from being merged.

**Comment by glguy** [comment]
We'll want to re-run CSV generation with the ASSIST generated data in place. I'm not exactly sure how that works, yet.

**Comment by glguy** [comment]
OK, I found the issue. Don't use `T` in your dateTime literals, SADL doesn't like that. Use a *space* instead.

**Comment by AbhaMoitra** [comment]
@glguy : Thank you very much! Here is what I see - want to confirm it with you.
In SADL I have has generatedAtTime "2017-03-23 10:03:16"
In OWL I see (note -4:00)
<j.0:generatedAtTime rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime"
    >2017-03-23T10:03:16-04:00</j.0:generatedAtTime>
In csv I see Thu Mar 23 10:03:16 EDT 2017
In SemTK I see 2017-03-23T10:03:16-04:00

**Comment by glguy** [comment]
@AbhaMoitra as you pointed out the example wasn't specifying a timezone. Unfortunately SADL is rendering the results out in the local timezone, but I've updated the input to use UTC explicitly. We'll want people to specify timezones in their inputs times so that the semantics of their files don't depend on the current location, so this will make for an even better example.

**Review by russell-d-e — APPROVED** [review]

**Pull request merged**